src/app.css

```
@@ -4,6 +4,7 @@
 @import '@fontsource/open-sans/700.css';
 @import 'tailwindcss';
 @config '../tailwind.config.ts';
+@plugin '@tailwindcss/typography';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)